### ISSUE TITLE
sample shape is not as expected

### ISSUE THREAD

**Issue by ferrine** [pymc-devs/pymc#1561]
Hi! I got stuck with that problem. It forces me to reshape sampled var and assumes my knowledge about true shape.
minimal example to reproduce the problem
```python
>>> m = pm.Model()
>>> n = m.Var('n', pm.Normal.dist(shape=(1, 1))
>>> n.random()
array([ 1.3160956])
>>> n.tag.test_value.shape == (1, 1)
True
>>> n.random().shape == (1, 1)
False
```
I use `random()` for creating initial weights in network and I need them working properly

**Comment by twiecki** [comment]
Yep, that seems wrong. It's likely we're doing a ravel() somewhere. Does it work for other shapes?

**Comment by twiecki** [comment]
Closed by #1585.
